.github/workflows/stellar-rpc.yml

```diff
@@ -77,19 +82,25 @@ jobs:
 
       - run: |
           rustup target add ${{ matrix.rust_target }}
+          rustup toolchain install stable-${{ matrix.rust_target }}
+          rustup default stable-${{ matrix.rust_target }}
+          rustup set default-host ${{ matrix.rust_target }}
           rustup update
+
       - uses: stellar/actions/rust-cache@main
       - run: make build-libs
         env:
           CARGO_BUILD_TARGET: ${{ matrix.rust_target }}
 
       - name: Build Stellar RPC reproducible build
         run: |
-          go build -trimpath -buildvcs=false ./cmd/stellar-rpc
+          go build -trimpath -ldflags="-v -linkmode=external" -buildvcs=false ./cmd/stellar-rpc
           ls -lh stellar-rpc
           file stellar-rpc
         env:
           CGO_ENABLED: 1
+          CGO_LDFLAGS: -v
+          GOTOOLCHAIN: local
```

Comment:
Is this for the windows build?

Reply:
yeah, buncha debugging commits - reverted